docker-compose.yml

```diff
@@ -435,14 +435,15 @@ services:
     ports:
       - "${QUOTE_SERVICE_PORT}"
     environment:
-      # OTEL_EXPORTER_OTLP_TRACES_ENDPOINT # Not working for PHP
+      - OTEL_EXPORTER_OTLP_TRACES_ENDPOINT=http://otelcol:4318/v1/traces
+      #- OTEL_EXPORTER_OTLP_ENDPOINT=http://otelcol:4318
+      - OTEL_PHP_AUTOLOAD_ENABLED=true
       - QUOTE_SERVICE_PORT
       - OTEL_SERVICE_NAME=quoteservice
-      - OTEL_EXPORTER_OTLP_ENDPOINT=http://otelcol:4318
       - OTEL_TRACES_SAMPLER=parentbased_always_on
       - OTEL_TRACES_EXPORTER=otlp
       - OTEL_EXPORTER_OTLP_TRACES_PROTOCOL=http/protobuf
-      - OTEL_PHP_TRACES_PROCESSOR=simple
+      - OTEL_PHP_TRACES_PROCESSOR=batch
```

Comment:
is `batch` the default now? If so we can remove this environment variable

Reply:
yes, batch is default. I have also removed other env vars that were using default values